src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -1776,7 +1791,9 @@ protected function _WriteDataToDatabase(&$oFields, &$oData, $bDataFromDatabase =
         // we need this because we use a redirect later and would not see the error message, we need to change this later to a nice error_logging or something
         if ($bWasInserted) {
             $this->LoadDataFromDatabase();
-            if ($this->IsQueryLoggingAllowed()) {
+            /** @var  $migrationRecorderStateHandler \ChameleonSystem\DatabaseMigrationBundle\Bridge\Chameleon\Recorder\MigrationRecorderStateHandler*/
```

Comment:
The namespace here should be shortened (with an import/use).

Reply:
done